app/assets/javascripts/utils/course_utils.js

```diff
@@ -99,10 +99,11 @@ const CourseUtils = class {
       url: articleUrl,
       title: assignment.article_title,
       article_id: assignment.article_id,
-      language,
-      project,
+      language: language,
+      project: project,
```

Comment:
This is fine, but means the same thing as what was there before, which is es2015 shorthand notation: `{ foo }` is the same as `{ foo: foo }`.